### PR TITLE
add maple-fonts

### PR DESCRIPTION
**Pull request by pugaizai**
#859 
just messed up some things

### CODE REVIEW

**Comment by lleyton** [comment]
Thanks for your contribution, I'll review this tonight and merge if all looks well!

**Comment by madonuko** [comment]
Might also want to use `%setup -q`

File: anda/fonts/maple/maple-fonts.spec [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+cp %{S:3} .
+ 
+%build
+ 
+%install
+install -d %{buildroot}%{_ttfontsdir}
+# by default install command uses 755 umask
+install -m 644 *.ttf %{buildroot}%{_ttfontsdir}
+ 
+%post
```
Comment (lleyton) [inline anda/fonts/maple/maple-fonts.spec]: You shouldn't have to specify %post and %postun, since you aren't doing anything here anyways.

**Review by lleyton — CHANGES_REQUESTED** [review]
Works locally, see my other comment though. Once that's addressed, this is good to merge.

File: anda/fonts/maple/maple-fonts.spec (hunk 1 above)
Comment (pugaizai) [reply anda/fonts/maple/maple-fonts.spec]: done

**Review by lleyton — APPROVED** [review]

**Comment by lleyton** [comment]
Congratulations on your first Terra package! We can't wait to see what else you might contribute <3

Your submission will shortly be built and uploaded to the production repositories.

**Pull request merged**